test/Web/ScottySpec.hs

```diff
@@ -56,6 +56,9 @@ spec = do
           it ("properly handles extra slash routes for " ++ method ++ " requests") $ do
             makeRequest "//scotty" `shouldRespondWith` 200
 
+        withApp (route "/:paramName" $ param "paramName" >>= text) $ do
+          it ("captures route parameters for " ++ method ++ " requests when parameter matches its name") $ do
+            makeRequest "/:paramName" `shouldRespondWith` ":paramName"
```

Comment:
I don't think this is the correct behaviour.  

```haskell
makeRequest "/:x" `shouldRespondWith` "x" 
```
  instead, without the colon.

Reply:
Hi sorry for the delayed response, I didn't notice your comments until now.
I believe the current test is correct as the colon is part of the value. From the point of view of the user the colon has no special meaning.